test_submission_schema/schemas/v1.json

```diff
@@ -0,0 +1,177 @@
+                    "type": "string",
+                    "description": "A brief description of the test plan."
+                },
+                "execution_id": {
+                    "type": "string",
+                    "description": "Execution ID for running the test plan."
+                }
+            }
+        },
+        "results": {
+            "type": "array",
+            "items": {
+                "type": "object",
+                "required": [
+                    "id",
+                    "test_description",
+                    "certification_status",
+                    "category_id",
+                    "outcome",
+                    "comments",
+                    "io_log",
+                    "duration",
+                    "type",
+                    "template_id"
+                ],
+                "properties": {
+                    "id": {
+                        "type": "string",
+                        "pattern": "^[a-zA-Z0-9_]+\\.[a-zA-Z0-9_]+\\.[a-zA-Z0-9_]+::[a-zA-Z0-9_]+/[a-zA-Z0-9_]+$",
+                        "description": "The unique identifier of the test case."
+                    },
+                    "test_description": {
+                        "type": "string",
+                        "description": "The description of the test case.",
+                        "maxLength": 80
+                    },
+                    "certification_status": {
+                        "type": "string",
+                        "description": "The certification status of the test case.",
+                        "enum": [
+                            "blocker",
+                            "non-blocker"
+                        ]
+                    },
+                    "category_id": {
+                        "type": "string",
+                        "pattern": "^[a-zA-Z0-9_]+\\.[a-zA-Z0-9_]+\\.[a-zA-Z0-9_]+::[A-Za-z0-9_\\-]+$",
+                        "description": "The unique identifier of the test case category."
+                    },
+                    "outcome": {
+                        "type": "string",
+                        "description": "The outcome of the test case.",
+                        "enum": [
+                            "passed",
+                            "failed",
+                            "skipped"
+                        ]
+                    },
+                    "comments": {
+                        "type": "string",
+                        "description": "Any additional comments or observations about the test case."
+                    },
+                    "io_log": {
+                        "type": "string",
+                        "description": "The stdout and stderr for the test case."
+                    },
+                    "duration": {
+                        "type": "number",
+                        "minimum": 0,
+                        "description": "The duration of the test case in milliseconds."
```

Comment:
Should we go down milliseconds? Checkbox stores this value in seconds (example: `duration: 6.5551793575286865`)

Reply:
changed to seconds